app/assets/scripts/components/project/sec-panel/class-analytics-chart.js

```diff
@@ -139,34 +138,30 @@ function ClassAnalyticsChart(props) {
         />
       </ChartContainer>
       <Summary>
-        {bounds && (
-          <ClassItem bounds={bounds}>
-            <p> </p>
-            <Prose size='small'>CLASS NAME</Prose>
-            <Prose size='small' className='percent'>
-              AREA KM2
-            </Prose>
-            <Prose size='small' className='percent'>
-              %
-            </Prose>
-          </ClassItem>
-        )}
+        <ClassItem>
+          <p> </p>
+          <Prose size='small'>CLASS NAME</Prose>
+          <Prose size='small' className='percent'>
+            AREA KM2
+          </Prose>
+          <Prose size='small' className='percent'>
+            %
+          </Prose>
+        </ClassItem>
         {Object.values(checkpoint.classes).map(
           (c, i) =>
             checkpoint.analytics[i] && (
-              <ClassItem key={c.name} bounds={bounds}>
+              <ClassItem key={c.name}>
```

Comment:
```suggestion
              <ClassItem key={c.name} totalArea={totalArea}>
```
reinstate to ensure that the middle column only appears on the Class Distribution chart, not the F1 score chart